src/data.rs

```diff
@@ -287,6 +304,7 @@ mod printing {
                 Visibility::Restricted(ref path) => {
                     tokens.append("pub");
                     tokens.append("(");
+                    tokens.append("in");
```

Comment:
That's fine right? Even if there were such a file, syntex would parse the before and after into the same AST.